src/api/mapping.ts

```diff
@@ -11,13 +11,16 @@ const exec = util.promisify(cp.exec)
 
 mappingRouter.post('/', (req, res) => {
   const domainKeys = getMappings()
-  const sortedKeys = domainKeys.sort((a,b) =>parseInt(a.port) - parseInt(b.port))
-  let portCounter = 3002
-  if(req.body.port === ''){
-    sortedKeys.forEach((e) => {
-      if(e.port === portCounter.toString()) portCounter += 1
-    })
+  const getNextPort = (map, start=3002):number =>{
```

Comment:
this helper function does not need closure (therefore, does not need to be defined inside the callback handler). please move outside.